docs/docs/catalogues/blocks/index.ipynb

```diff
@@ -25,17 +25,17 @@
     "\n",
     "<div class=\"grid cards\" markdown>\n",
     "    \n",
-    "-   <a href=\"/catalogues/blocks/table\"><img style=\"width:100%;\" src=\"../../img/catalogues/table.png\"></a>\n",
+    "-   __[Table](/catalogues/blocks/table)__\n",
```

Comment:
What was the rationale behind switching titles/images? We had them in the more common order (newspaper/print/website cards)

Reply:
I wasn't sure if this was intentional. I'd first read these pages on mobile, and found the styling on [blocks](https://docs.datapane.com/catalogues/blocks/) of img/title to be a bit confusing, but clearer on [components](https://docs.datapane.com/catalogues/components/) where it's tile/img. I think img/title would be fine, but could do with a bit more styling to associate them - max-height the image and group them in a visual container perhaps. Or just swap their order on mobile with css?